src/lib/videotagging_extensions/detection.js

```diff
@@ -104,14 +88,38 @@ function Detection(videotagging, visitedFrames) {
                 if (err) {
```

Comment:
If there is an error we still want the export to continue? As of right now it will call the callback method with the error but still continue with the export process.